fabtests/benchmarks/benchmark_shared.c

```diff
@@ -161,6 +161,10 @@ static int bw_rx_comp()
 				   opts.transfer_size);
 		if (ret)
 			return ret;
+
+		ret = ft_check_cq_data();
+		if (ret)
+			return ret;
```

Comment:
The benchmarks are performance focused tests (arguably).  Adding checks to the test is not desirable.